docs/docs/ports.md

```diff
@@ -16,7 +16,7 @@ Port configuration for applications in Marathon can be confusing and there is [a
 
 *HOST networking*: `HOST` networking is used by non-Docker Marathon applications and Docker applications that use `HOST` mode networking. In this mode, applications bind directly to one or more ports on the host machine.
 
-*portMapping*: A _port mapping_ is necessary for Docker applications that use `BRIDGE` mode networking and is a tuple containing a host port, container port, service port and protocol. Multiple _port mappings_ may be specified for a Marathon application.
+*portMapping*: A _port mapping_ is used to define ports that should be considered as part of a resource offer. It is necessary for Docker applications that use `BRIDGE` mode networking and is a tuple containing a host port, container port, service port and protocol. Multiple _port mappings_ may be specified for a Marathon application.
```

Comment:
@ssk2 sorry for being picky, but the logic is wrong: it must be "... is used to define ports ... in BRIGDE mode". That's different from "... is used to define ports ... <FULLSTOP>. It is necessary ... for BRIDGE mode".